src/simulator/RangeSensor.tsx

```diff
@@ -83,8 +85,11 @@ const RangeSensor = ({
           placement="top"
           label={valueText}
           isOpen={minimised ? showTooltip : false}
+          onOpen={() => sliderRef.current!.removeAttribute("aria-describedby")}
```

Comment:
This feels risky, presumably this is react-managed DOM.

Reply:
Is there any value we can pass in for the prop that will achieve this?

Reply:
Different approach in 40b13963703dc6d07231a0b1483d9aaa6b3d019c. Instead of removing it at runtime (which might break reconciliation), wrap the component and ignore it.